core/include/kernel/user_access.h

```diff
@@ -10,25 +10,8 @@
 #include <tee_api_types.h>
 #include <types_ext.h>
 
-#ifdef CFG_WITH_USER_TA
 TEE_Result copy_from_user_private(void *kaddr, const void *uaddr, size_t len);
 TEE_Result copy_from_user(void *kaddr, const void *uaddr, size_t len);
-#else
-static inline TEE_Result copy_from_user_private(void *kaddr __unused,
```

Comment:
I'd prefer keeping these, I believe that the corresponding .c file shouldn't be built in this case.